src/nvim/fileio.c

```diff
@@ -706,10 +706,9 @@ readfile (
      * still be running, don't move the cursor to the last line, unless
      * always using the GUI.
      */
-    if (read_stdin) {
-      mch_msg(_("Nvim: Reading from stdin...\n"));
-    } else if (!read_buffer)
+    if ((!read_stdin) && (!read_buffer)) {
       filemess(curbuf, sfname, (char_u *)"", 0);
+    }
```

Comment:
I don't know why it was added before, but I just made condition, which will be triggered in same conditions as before.

There was "if read stdin then throw warning, otherwise, if not read buffer, then filemess".
I did "if not read stdin and not read buffer, then filemess".

So, `filemess` will be triggered in exact conditions as before: when "not read_stdin" and "not read_buffer"...